src/Orleans/Serialization/BuiltInTypes.cs

```diff
@@ -1850,6 +1852,7 @@ internal static void SerializeInvokeMethodRequest(object obj, ISerializationCont
         internal static object DeserializeInvokeMethodRequest(Type expected, IDeserializationContext context)
         {
             int iid = context.StreamReader.ReadInt();
+            ushort iVersion = context.StreamReader.ReadUShort();
```

Comment:
I don't think we ever guaranteed that two versions of Orleans were compatible, why starts now?